docs/user-guide/global-configurations/chart-repo.md

```diff
@@ -0,0 +1,60 @@
+# Chart Repository
+
+This feature allows you to add more chart repositories to Devtron. Once added they will be available in the Discover section of the Chart Store. 
```

Comment:
This feature allows you to add chart repositories to Devtron. You can add as many chart repositories as you require. Once added, they will be available in the Discover section of the 'Chart Store'.